test/catalog/01/imported-xspec-uri.xspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<t:description expand-text="1"
+	query="x-urn:test:do-nothing"
+	query-at="do-nothing.xqm"
+	schematron="do-nothing.sch"
+	stylesheet="do-nothing.xsl"
+	xml:base="../../"
+	xmlns:im="x-urn:test:imported-xspec-uri"
+	xmlns:t="http://www.jenitennison.com/xslt/xspec"
+	xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+	<t:variable name="im:xspec-uri-in-global-var"
+		select="$t:xspec-uri"/>
+
+	<t:scenario label="$x:xspec-uri defined and checked in imported file">
+		<t:call function="true" />
+		<!-- Assertion fails when this test file runs and
+			passes when the importing file, xspec-uri.xspec, runs.
+			The regular expression is designed to match files named
+			catalog-01_xspec-uri.xspec and xspec-uri.xspec.
+		-->
+		<t:expect label="should be top-level .xspec file that is running"
+			test="matches($im:xspec-uri-in-global-var, '[_/]xspec-uri\.xspec$')" />
```

Comment:
Fair enough. I updated it in 75a51d2facb07b24069ef197fb7cd91d7ead50ff to test each of the two expected filenames separately using `ends-with`.

It could be done in a fancier way (and more scalable, but we only have 2 filenames here) like this
```
test="('/catalog-01_xspec-uri.xspec','/xspec-uri.xspec')
=> filter(ends-with($im:xspec-uri-in-global-var,?))
=> exists()"/>
```
but that won't work with Saxon 9.9 HE. If you'd like me to update this after you remove testing with Saxon 9, let me know.

Reply:
Thanks. Noted in https://github.com/xspec/xspec/issues/1826#issue-2032559204.